src/components/utils/caseUtils.ts

```diff
@@ -200,19 +208,38 @@ export function useDownloadUtils() {
         const cases = selectedElements.filter(
             (element) => element.type === ElementType.CASE
         );
-        await exportCases(
-            cases,
-            format,
-            formatParameters,
-            handleCaseExportError
-        );
-        if (cases.length !== selectedElements.length) {
-            snackInfo({
-                messageTxt: buildPartialDownloadMessage(
+        let message: string = '';
+
+        try {
+            const controller = new AbortController();
+            setAbortController(controller);
+
+            for (const c of cases) {
+                await exportCase(c, format, formatParameters, controller);
+            }
+        } catch (error: any) {
+            if (error.name === 'AbortError') {
+                message = intl.formatMessage({
+                    id: 'download.stopped',
+                });
+            }
+        } finally {
+            setAbortController(undefined);
+
+            if (
+                message.length === 0 &&
+                cases.length !== selectedElements.length
+            ) {
+                message += buildPartialDownloadMessage(
                     cases.length,
                     selectedElements
-                ),
-            });
+                );
+            }
+            if (message.length > 0) {
+                snackInfo({
```

Comment:
A small bug arrives after migrate the new version common ui, i.e. the snackInfo without headerId do not show the snack bar. headerId is optional as declared in the type definition file.
This bug will be corrected later in common ui

Reply:
Indeed. But I merge anyway : this is a priority correction and it still work despite the bug in main. The snackbar is simply not displayed at the end of the download (or its cancellation).